ambry-tools/src/main/java/com.github.ambry/account/AccountUpdateTool.java

```diff
@@ -248,18 +238,70 @@ private static AccountService getHelixAccountService(String zkServer, String sto
     helixConfigProps.setProperty(HelixPropertyStoreConfig.HELIX_PROPERTY_STORE_PREFIX + "root.path", storePath);
     helixConfigProps.setProperty(HelixAccountServiceConfig.BACKUP_DIRECTORY_KEY, backupDir);
     VerifiableProperties vHelixConfigProps = new VerifiableProperties(helixConfigProps);
-    return new HelixAccountServiceFactory(vHelixConfigProps, new MetricRegistry()).getAccountService();
+    accountService = new HelixAccountServiceFactory(vHelixConfigProps, new MetricRegistry()).getAccountService();
+  }
+
+  /**
+   * Edit accounts in a text editor and upload them to zookeeper.
+   * @param accountNames the name of the accounts to edit.
+   * @throws IOException
+   * @throws InterruptedException
+   */
+  void editAccounts(Collection<String> accountNames) throws IOException, InterruptedException {
+    Path accountJsonPath = Files.createTempFile("account-update-", ".json");
+    JSONArray accountsToEdit = new JSONArray();
+    accountNames.stream()
+        .map(accountName -> accountService.getAccountByName(accountName).toJson(false))
+        .forEach(accountsToEdit::put);
+    try (BufferedWriter writer = Files.newBufferedWriter(accountJsonPath)) {
+      accountsToEdit.write(writer, 2, 0);
+    }
+    ToolUtils.editFile(accountJsonPath);
+    System.out.println("The following account metadata will be uploaded:");
+    try (Stream<String> lines = Files.lines(accountJsonPath)) {
+      lines.forEach(System.out::println);
+    }
+    if (ToolUtils.yesNoPrompt("Do you want to update these accounts?")) {
+      updateAccountsFromFile(accountJsonPath.toAbsolutePath().toString());
+    } else {
+      System.out.println("Not updating any accounts");
+    }
+  }
```

Comment:
Can we delete the temp file here?  Or want to leave it for further editing?

Reply:
I will delete it here, since the changes will be in the local backup file